src/cf-layout/src/molecules/heroes.less

```diff
@@ -62,133 +66,121 @@
         } );
     }
 
-    &_subhead,
-    &_cta:not( .btn ) {
+    &_subhead {
         // Not using the `.heading-3()` mixin here because we want the weight
         // to remain Regular on smaller screens.
         .u-webfont-regular();
 
-        font-size: @size-iv;
+        font-size: @size-iii;
         line-height: 1.25;
 
-        .respond-to-min( @bp-med-min, {
-            font-size: @size-iii;
-        } );
-    }
-
-    &_subhead:last-child {
-        margin-bottom: 0;
-    }
-
-    &_cta {
-        &:not( .btn ) {
-            border-bottom-width: 1px;
-        }
-
-        .respond-to-min( @bp-sm-min, {
-            margin-bottom: 0;
+        .respond-to-max( @bp-sm-max, {
+            font-size: @size-iv;
         } );
     }
 
     &_image-wrapper {
         .grid_column( 1, 1 );
 
+        overflow: hidden;
+
         .respond-to-min( @bp-sm-min, {
             .grid_column( 5, 12 );
 
-            // Keep the image vertically centered
```

Comment:
It's still applicable to everything from L:90-L:98, but I don't know that it's all that useful. The spec calls out the vertically centered alignment. If you think it still is I can put it back.